LICENSE

```diff
@@ -5,10 +5,10 @@ Redistribution and use in source and binary forms, with or without modification,
 are permitted provided that the following conditions are met:
 
     1. Redistributions of source code must retain the above copyright notice,
-       this list of conditions and the following disclaimer.
+       this list of conditions, and the following disclaimer.
 
     2. Redistributions in binary form must reproduce the above copyright
-       notice, this list of conditions and the following disclaimer in the
+       notice, this list of conditions, and the following disclaimer in the
```

Comment:
I was reluctant to accept this change until I looked for more info and found this 😂 :
![image](https://user-images.githubusercontent.com/37369/40868648-1b0809a8-65e5-11e8-993d-948944eca227.png)

source: http://themoosebody.tumblr.com/post/10317138331/unlikelywords-why-the-oxford-comma-is